app/controllers/tabs/tabs_controller.rb

```diff
@@ -78,6 +78,13 @@ def update
     end
   end
 
+  def sort
+    params[:order].each do |_key, value|
+      Tab.find(value[:id]).update_column(:pos, value[:position])
```

Comment:
Probably gonna need a better name than `value`. I am not sure what is it until I read the javascript code.